documentation/SupportingDocuments/API-Purpose-Inventory.md

```diff
@@ -0,0 +1,5 @@
+# API-Purpose-Inventory
```

Comment:
I think the purposes associated with each API are discussed by the privacy teams. Maybe we need them here to give their input...